atd-vze/src/routes.js

```diff
@@ -71,70 +42,6 @@ const routes = roles => [
   { path: "/dev/theme", exact: true, name: "Theme", component: Colors },
```

Comment:
any reason not to get rid of these other pieces? Notifications, Theme, Dev...?

Reply:
doneee